Scuti/Scripts/UI/Account/UserCredentialsForm.cs

```diff
@@ -19,13 +19,56 @@ public class Model : Form.Model
         {
             public string Email;
             public string Password;
+
             public string fullName;
+            public string phoneNumber;
+            public string gender = "Male";
+            public DateTime birthDay;
         }
 
+        // For email password
         [SerializeField] InputField emailInput;
         [SerializeField] InputField passwordInput;
         [SerializeField] Button registerButton;
 
+        //For info User
+        [SerializeField] InputField NameInput;
+        [SerializeField] Dropdown GenderDropDown;
+        [SerializeField] Dropdown YearDropDown;
+
+        #region UserDetails
+
+        protected override void Awake()
+        {
+            base.Awake();
+
+            Debug.Log("Opened Form");
+
+            YearDropDown.ClearOptions();
+
+            var year = DateTime.Now.Year;
+            List<Dropdown.OptionData> years = new List<Dropdown.OptionData>();
+            for (var i = 0; i < 100; i++)
+            {
+                years.Add(new Dropdown.OptionData(year.ToString(), null));
+                year--;
+
+            }
+            YearDropDown.AddOptions(years);
+        }
+
+        public override void Open()
+        {
+            base.Open();
+            Data.birthDay = new DateTime(2000, 2, 15);
+            Data.gender = "Male";
+            Data.phoneNumber = "3214567878"; //For testing
```

Comment:
Should we remove this since it is for testing?  @Jpablogs8   Probably Gender should be blank as well by default